backend/src/main/java/ubb/courses/backend/controllers/LessonController.java

```diff
@@ -0,0 +1,46 @@
+package ubb.courses.backend.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import ubb.courses.backend.DTOs.LessonDTO;
+import ubb.courses.backend.services.LessonService;
+
+import java.util.Set;
+
+@Controller
+@RequestMapping("/api/lesson")
+public class LessonController implements CrudController<LessonDTO>{
+
```

Comment:
Lessons are a different pr. We try to keep Pull Requests as small as possible for better code reviews and to divide tasks between the others. So please sparate your task into smaller PRs